File: ruby/multiples_of_three_and_five.rb
def multiples(x, y)
  sum = 0
  num = 0
  until num == 1000 do
    if num % 3 == 0 || num % 5 == 0
      sum += num
    end
    num += 1
  end
  sum
end

puts multiples(3, 5)

File: javascript/multiplesOfThreeAndFive.js
let multiples = (x, y) => {
  let sum = 0;
  for (let i = 0; i < 1000; i++) {
    if (i % x === 0 || i % y === 0) { sum += i; }
  }
  return sum;
};

let z = multiples(3, 5);

console.log(z);
